src/user_guide/003_Auth.md

```diff
@@ -1,56 +1,208 @@
+
+`Unit master token` and `unit user token` are prepared as administrative tokens for doing these.
+
+Please refer to [Unit User](../unit-administrator/Unit-User) for details.
+
+### Token type
+
+Many tokens have been introduced to access data. The table below shows the summary.
+
+| Token type | Description | Expiration date |
+|----|----|----|
+| Cell Local Token | Access token valid in the cell that issued the token | 1 hour |
+| Trans-cell token | Access token valid in the target cell specified by the issued cell | 1 hour |
+| Refresh Token | Token used to refresh an already issued token | 24 hours |
+|Unit Master Token |Administrative token used for initial settings such as cell creation immediately after unit construction, development and testing |-|
+| Unit user token | Administrative token used for operations such as cell creation |-|
+
+-------------------------------------------------------------------------------
+
+## RBAC (Role Based Access Control)
+
+In the previous section, it was stated that Role is determined by Account in the authenticated cell and the relationship between cells. Personium has a mechanism of RBAC (Role Based Access Control) that controls how much data access is allowed by this Role.
+
+A user controls access by setting which Role can perform what operation (ACL) for each resource.
+
+![ACL](assets/auth/acl-role.png)
+
+Please refer to [Access Control Model](../apiref/006_Access_Control.md) for details.
+
+-------------------------------------------------------------------------------
+
+## App authorization
+
+So far, for simplicity we have described a data access method that does not pass through the application. However, when the data accessor actually accesses the data, it basically does not directly access the cell and transfers the data access to the application.
+
+Also, in order to utilize the data on PDS, it is necessary to have abundant applications below.
+
+-App that provides data
+-App that uses data
+
+To do so, it is necessary to utilize open APIs that allow not only one business operator to develop and provide applications but also other business operators. This section describes the outline of application authorization under such assumptions.
+
+### OAuth in PDS
+
+Appropriate delegation of authority is required to allow apps that take various forms of provision to access data securely.
+Personium uses OAuth, which is a framework for app authorization and delegation, to achieve these.
+
+As described above, PDS Personium has the function of the authorization server in the cell of each data subject. If you apply to OAuth characters in this case, it will be as shown in the following figure.
```

Comment:
```suggestion
As described above, Personium has the function of the authorization server in each cell. If you apply to OAuth characters in this case, it will be as shown in the following figure.
```